### DISCUSSION TITLE
Can anyone help me?

### DISCUSSION THREAD

**Discussion by SMK-SonHR** [exult/exult#585] [General]
Can anyone explain or help me with this bug?

![이미지_1047](https://github.com/user-attachments/assets/f6cfe420-9297-4b4a-8e20-37e07f1fc56c)

When you go to Fire Island, that message pops up and it becomes extremely slow. 
The resolution is very high at 2048*1200. 
If you lower the resolution, the slowdown will disappear, but is there an explanation or solution for that phrase?

**Comment by DominusExult** [comment]
I can't reproduce.
Can you give us more information?
- exact location (coordinates) you notice this
- graphics settings (scaler, game area, resolution)

And most importantly, can you reproduce with current unmodified Exult snapshots (not the Korean modded Exult)?
